### DISCUSSION TITLE
OpenSSL 3.3.2 is compatible with PostGre SQL 14.13?

### DISCUSSION THREAD

**Discussion by SonalEric** [openssl/openssl#25969] [Q&A] [answered]
Hello All,

Please if you anyone can confirm if OpenSSL 3.3.2 is compatible with PostGre SQL 14.13?

Regards
Sonal Sharma

**Comment by nhorman** [comment, marked as answer]
Thats really a question for the PostgreSQL project.  Compatibility will depend on what API calls they make into the OpenSSL library.  Looking at the PostgreSQL documentation, they state the minimum required version for 14.3 is openssl 1.0.1.  They make no statement regarding 3.x compatibility.
